Sources/AWSLambdaRuntime/LambdaRuntimeClientError.swift

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-package struct LambdaRuntimeError: Error {
+package struct LambdaRuntimeClientError: Error {
```

Comment:
Why the rename? I don't think the `Client` adds more info here.

Reply:
@fabianfett  because this class only deals with errors from the HTTP Client (`LambdaRuntimeClient`) and is internal to the library.

The new `LambdaRuntimeError` is public, exposed to users, and represent a user error when using the Runtime, in this case, not respecting the singleton